content/quickstart.md

```diff
@@ -39,8 +83,74 @@ Under construction.
 
 ## Loading your model
 
-Under construction.
+#### First time
+
+1- open the ``model`` menu
+
+2- load model
+
+3- open ``.vrm`` file
+
+#### Loading a used model
+
+1- open ``preset`` menu
+
+2- select preset and hit ``load``
+
+## Tracking
+
+### OpenSeeFace
+
+Runs using your webcam on laptop/pc. Source code provided by [emilianavt](https://github.com/emilianavt/OpenSeeFace)
+
+### IFacialMocap
+
+Uses the 3D tracking capabilities of Apple Inc devices to provide optimal tracking data from your phone. Use if your model has

```

Comment:
"Inc" should have a "." to indicate that it is an abbreviation

> Use if your model has perfect sync.

Should be "Use this if ..."